### PR TITLE
Add [output] audio_dir to store recordings separately from transcripts

### PR DESCRIPTION
**Pull request by aaugustin**
Audio is a disposable cache once transcription succeeds, while transcripts and summaries are the real output worth syncing across devices and backing up. Storing the audio elsewhere avoids overloading a sync or backup system with uncompressed audio files.

audio_dir defaults to empty (same as dir, current behavior) and, when set, keeps the audio directory in sync with the standard output directory's timestamp name and title-slug rename across the record, transcribe, summarize, resume, and cleanup flows.

### CODE REVIEW

**Comment by aaugustin** [comment]
For clarity: the code is half Claude half me; yes it started with a prompt, but I reworked heavily to bring to PR to an acceptable level of quality.

File: src/ownscribe/pipeline.py [hunk 1]
```diff
@@ -555,7 +573,14 @@ def run_resume(config: Config, directory: str) -> None:
...
         raise SystemExit(1)
 
+    # First look for an audio file in the target directory, since the user is
+    # explicitly resuming there. If none is found and a separate directory is
+    # configured for audio, search that directory as well.
     audio = _find_audio(dir_path)
+    if audio is None:
+        candidate = config.output.resolved_audio_dir / dir_path.name
+        if candidate != dir_path and candidate.is_dir():
+            audio = _find_audio(candidate)
```
[the remark this answers is not in the document]
Comment (aaugustin) [reply src/ownscribe/pipeline.py:603]: This case is already covered by the condition `if candidate != dir_path` two lines below and explained in the comment two lines above.

File: src/ownscribe/pipeline.py [hunk 2]
```diff
@@ -382,12 +401,12 @@ def run_summarize(config: Config, transcript_file: str) -> None:
...
-        try:
-            out_dir.rename(new_dir)
-            out_dir = new_dir
-        except Exception:
-            logging.getLogger(__name__).warning("Could not rename output directory", exc_info=True)
+        audio_dir = config.output.resolved_audio_dir / out_dir.name
+        out_dir, old_out_dir = _rename_output_dir(out_dir, title_slug), out_dir
+        # Rename audio_dir only if it's separate from out_dir
+        # and if out_dir was successfully renamed.
+        if audio_dir != old_out_dir and out_dir != old_out_dir:
```
[the remark this answers is not in the document]
Comment (aaugustin) [reply src/ownscribe/pipeline.py]: The case when `output.audio_dir` is unset is covered by `if audio_dir != old_out_dir`.

Targeting and renaming would still work fine when the summarizing a transcript outside the configured output tree as far as I can tell.

File: src/ownscribe/cli.py [hunk 3]
```diff
@@ -274,20 +275,27 @@ def cleanup(
...
         if cache:
             targets.append(("Cache", _CACHE_DIR))
         if output:
             targets.append(("Output", output_dir))
+            if audio_dir != output_dir:
+                targets.append(("Audio", audio_dir))
     else:
-        # Interactive: prompt for each directory
-        for label, path in [
+        targets = [
```
Comment (paberr) [inline src/ownscribe/cli.py]: Here we have a clash in variable naming. `targets` is now the name for the list of things to prompt for **and** the name for the variable storing where the user replied "yes".
Maybe rename the first to `candidates` or so?

File: src/ownscribe/pipeline.py [hunk 4]
```diff
@@ -496,21 +515,20 @@ def _do_transcribe_and_summarize(
...
-                out_dir = new_dir
-            except Exception:
-                logging.getLogger(__name__).warning("Could not rename output directory", exc_info=True)
+            audio_dir = audio_path.parent
+            out_dir, old_out_dir = _rename_output_dir(out_dir, title_slug), out_dir
+            # Rename audio_dir only if it's separate from out_dir
+            # and if out_dir was successfully renamed.
+            if audio_dir != old_out_dir and out_dir != old_out_dir:
+                audio_dir = _rename_output_dir(audio_dir, title_slug)
+            audio_path = audio_dir / audio_path.name
```
Comment (paberr) [inline src/ownscribe/pipeline.py:540]: If I see correctly, this will break the old behaviour:
1. Assume `keep_recording = false` and no separate audio directory, but we have a `title_slug`.
2. Then, `audio_dir` is only assigned to `audio_path.parent`, which is the pre-rename folder.
3. So, later on `audio_path.exists()` will return false.

In that case, `audio_dir` should become `out_dir`. The logic is currently a bit messy, maybe we can clean it up a bit?

File: src/ownscribe/pipeline.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (paberr) [reply src/ownscribe/pipeline.py]: Hm... I think Co-Pilot is right in the following case:

When you call `ownscribe summarize ~/notes/foo/transcript.md`, `audio_dir` is unset and `resolved_audio_dir` falls back to `resolved_dir`, which would try to rename `~/ownscribe/foo`.
The guard only helps if the transcript is inside the output tree.

Possible fix: only do the second rename if `config.output.audio_dir` is set and the directory exists.

File: src/ownscribe/pipeline.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (paberr) [reply src/ownscribe/pipeline.py:603]: see my comment above, on a second look, I think CoPilot is correct

**Review by paberr — CHANGES_REQUESTED** [review]
Thanks a lot for the work, this is a good addition.
I personally just have `keep_recording = false` all the time, but I can see how you might want to keep it but in a different directory.

I looked through the PR and found a few edge cases. After addressing them, it should be ready!

File: src/ownscribe/cli.py (hunk 3 above)
Comment (aaugustin) [reply src/ownscribe/cli.py]: Oops. You're right.

File: src/ownscribe/pipeline.py (hunk 4 above)
Comment (aaugustin) [reply src/ownscribe/pipeline.py:540]: Yes, good catch.

Comment (aaugustin) [reply src/ownscribe/pipeline.py:540]: I'm straightening the logic with clearer conditions and comments.

File: src/ownscribe/pipeline.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (aaugustin) [reply src/ownscribe/pipeline.py]: Yes, I had missed the fact that you can summarize an arbitrary file, even if it's outside of ownscribe's output tree.

**Comment by aaugustin** [comment]
I believe I have addressed all your comments, which were correct.

Also, I improved the branch that deletes the recording after transcription: I'm also deleting the directory if it's empty, which will be the standard case when storing audio in a separate directory.

**Comment by paberr** [comment]
Thanks for the quick turnaround on the review comments! Rather than another review round-trip, I pushed a small fixup commit (50db237) directly:

- `audio_path.parent.unlink()` can't remove a directory (`PermissionError` on macOS, `IsADirectoryError` on Linux), so every `keep_recording=false` run with a separate `audio_dir` crashed right after deleting the recording — now `rmdir()`.
- The two new `TestDoTranscribeAndSummarize` tests didn't set `output.audio_dir`, so they exercised the default colocated path instead of the feature; one of them was failing. They now configure the separate dir.
- Two edge-case guards while I was in there: resuming a directory that still holds its own recording (made before `audio_dir` was configured) now follows the out_dir rename instead of logging a spurious warning and leaving a stale path; and `summarize` on a transcript outside the output tree no longer renames a same-named, unrelated directory under `audio_dir`.

Full suite is green locally (256 passed). Merging once CI passes — thanks for the contribution!

**Review by paberr — APPROVED** [review]
All review feedback addressed; remaining issues fixed directly on the branch (see comment). LGTM.
